index.html

```diff
@@ -50,7 +51,258 @@ <h3 class="header-highlight-2">I’am a Software Developer</h3>
+                    </li>
+                    <li class="tags cd-tags cd-tag-3"><a href="#">Ruby</a></li>
+                  </ul>
+                </div>
+                <button type="button" class="button btn">
+                  <span class="cd-btn-txt">see project</span>
+                </button>
+              </article>
+              <article class="cards card-1">
+                <div class="card-body">
+                  <h2 class="card-title">profesional art printing date</h2>
+                  <p class="card-text">
+                    A daily selection of privately personalized reads; no
+                    accounts or sign-ups required. has been the industry's
+                    standard
+                  </p>
+                  <ul class="card-tags">
+                    <li class="tags cd-tag-1"><a href="#">html</a></li>
+                    <li class="tags cd-tag-2">
+                      <a href="#">bootstrap</a>
+                    </li>
+                    <li class="tags cd-tags cd-tag-3"><a href="#">Ruby</a></li>
+                  </ul>
+                </div>
+                <button type="button" class="button btn">
+                  <span class="cd-btn-txt">see project</span>
+                </button>
+              </article>
+              <article class="cards card-1">
+                <div class="card-body">
+                  <h2 class="card-title">profesional art printing date</h2>
+                  <p class="card-text">
+                    A daily selection of privately personalized reads; no
+                    accounts or sign-ups required. has been the industry's
+                    standard
+                  </p>
+                  <ul class="card-tags">
+                    <li class="tags cd-tag-1"><a href="#">html</a></li>
+                    <li class="tags cd-tag-2">
+                      <a href="#">bootstrap</a>
+                    </li>
+                    <li class="tags cd-tags cd-tag-3"><a href="#">Ruby</a></li>
+                  </ul>
+                </div>
+                <button type="button" class="button btn">
+                  <span class="cd-btn-txt">see project</span>
+                </button>
+              </article>
+              <article class="cards card-1">
+                <div class="card-body">
+                  <h2 class="card-title">profesional art printing date</h2>
+                  <p class="card-text">
+                    A daily selection of privately personalized reads; no
+                    accounts or sign-ups required. has been the industry's
+                    standard
+                  </p>
+                  <ul class="card-tags">
+                    <li class="tags cd-tag-1"><a href="#">html</a></li>
+                    <li class="tags cd-tag-2">
+                      <a href="#">bootstrap</a>
+                    </li>
+                    <li class="tags cd-tags cd-tag-3"><a href="#">Ruby</a></li>
+                  </ul>
```

Comment:
- [ ] You did a great job adding the list of skills that you needed for the project but you don't need to wrap them with tag a because they are not links. Kindly fix it.